tests/pass-dep/concurrency/linux-futex.rs

```diff
@@ -35,17 +35,6 @@ fn wake_nobody() {
     }
 }
 
-fn wake_dangling() {
```

Comment:
Would be nice if this were replaced with a test to check it still accepts "wake after free".